### DISCUSSION TITLE
How can you display an admonition icon without text?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#4562] [Q&A] [answered]
Curious to know what the easiest way would be to display admonition boxes with their icon, but no title.

Just using `""` won't work obviously, as it removes the icon too. Is a simple space enough or would I need something else (i.e. zero-width-space) to achieve this goal?

**Comment by facelessuser** [comment, marked as answer]
Honestly, probably something like this:

```
!!! note "&nbsp;"
    test
```

You need something there to force the height of the title bar, so `&nbsp;` works decent.

**Reply by Andre601** [reply]
This does the trick, thank you
